ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-commands.sh

```diff
@@ -6,18 +6,8 @@ set -o pipefail
 
 CONFIG="${SHARED_DIR}/install-config.yaml"
 
-case $((RANDOM % 8)) in
-0) AZURE_REGION=centralus;;
-1) AZURE_REGION=centralus;;
-2) AZURE_REGION=centralus;;
-3) AZURE_REGION=centralus;;
-4) AZURE_REGION=centralus;;
-5) AZURE_REGION=centralus;;
-6) AZURE_REGION=eastus2;;
-7) AZURE_REGION=westus;;
-*) echo >&2 "invalid Azure region index"; exit 1;;
-esac
-echo "Azure region: ${AZURE_REGION}"
+REGION="${LEASED_RESOURCE}"
```

Comment:
the leased resource is something like `<region>--<some number>` as in https://github.com/openshift/release/pull/14262/files#diff-5169f2a74d1497f38a44e9adc57f6993269a89c3ddf90ab01f5d1d114ef61e58R210

i think we need to transform the lease to region here.